online-shop/src/app/components/presentational/products-details-view/products-details-view.component.html

```diff
@@ -2,8 +2,8 @@
     <div class="product-details-header">
         <h1>{{product.name}}</h1>
         <div class="product-details-header__buttons">
-            <button [routerLink]="['/edit/',product.id]">Edit</button>
-            <button (click)="deleteProduct()" [routerLink]="['']">Delete</button>
+            <button *ngIf="isAdmin==='yes'" [routerLink]="['/edit/',product.id]">Edit</button>
```

Comment:
If you'll change the isAdmin type to be a boolean, there will be no need to have this check. You'll have just *ngIf="isAdmin" and *ngIf="!isAdmin"